### ISSUE TITLE
Youtube video taking long to download

### ISSUE THREAD

**Issue by sajaltiwari999** [maxtoncode/PostDownloader#1]

**Comment by cybermaxton** [comment]
yes it take little bit time because some resolutions do not download audio with video to solve that issue program download audio as well as video and merges the both for your better experience. In next update i may fix this issue. Untill wait :}
